R/tidyverse.R

```diff
@@ -143,19 +143,6 @@ use_tidy_dependencies <- function() {
   invisible()
 }
 
-#' @export
-#' @rdname tidyverse
```

Comment:
This never actually should've been here, since it wasn't a function designed for exclusive use of the tidyverse team.